CHANGELOG.md

```diff
@@ -60,6 +60,7 @@ And please only add new entries to the top of this list, right below the `# Unre
 - On Web, fix pointer button events not being processed when a buttons is already pressed.
 - **Breaking:** Updated `bitflags` crate version to `2`, which changes the API on exposed types.
 - On Web, handle coalesced pointer events, which increases the resolution of pointer inputs.
+- On Windows, port to `windows-sys` version 0.48.0.
```

Comment:
I don't think it's meaningful to users. Though the changelog entries have conflicts.

Reply:
Disagree; `windows-sys` version churn has been a frequent enough problem that's been resolved as of version 0.48.0 (at least for the time being). Knowledge that we are using this new version may be valuable to users trying to avoid `windows-sys` version churn.